core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -25,54 +25,39 @@
 public class OpenSearchFunctions {
   public void register(BuiltinFunctionRepository repository) {
     repository.register(match());
+    repository.register(match_phrase());
   }
 
   private static FunctionResolver match() {
     FunctionName funcName = BuiltinFunctionName.MATCH.getName();
+    // At most field, query, and all optional parameters
+    final int matchMaxNumParameters = 14;
+    return getRelevanceFunctionResolver(funcName, matchMaxNumParameters);
+  }
+
+  private static FunctionResolver match_phrase() {
+    FunctionName funcName = BuiltinFunctionName.MATCH_PHRASE.getName();
+    // At most field, query, and all optional parameters
+    final int matchPhraseMaxNumParameters = 5;
```

Comment:
ditto

Reply:
addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/48/commits/e6f7e2fc36f13e9904dab9ef106c49eec89ef25d